app/audit/AuditService.scala

```diff
@@ -71,7 +71,7 @@ class AuditServiceImpl @Inject()(
         logger.debug(s"[AuditService][sendEvent] successfully sent ${event.auditType}")
 
       case Failure(e) =>
-        logger.error(s"[AuditService][sendEvent] failed to send event ${event.auditType}", e)
+        logger.warn(s"[AuditService][sendEvent] failed to send event ${event.auditType}", e)
```

Comment:
Audit failures should be errors. They are critical to get right, they store historic data to be retrieved if something goes wrong.